src/Lexer.js

```diff
@@ -393,16 +399,8 @@ module.exports = class Lexer {
         continue;
       }
 
-      // strong
-      if (token = this.tokenizer.strong(src, maskedSrc, prevChar)) {
-        src = src.substring(token.raw.length);
-        token.tokens = this.inlineTokens(token.text, [], inLink, inRawBlock);
-        tokens.push(token);
-        continue;
-      }
-
-      // em
-      if (token = this.tokenizer.em(src, maskedSrc, prevChar)) {
+      // em & strong
+      if (token = this.tokenizer.emStrong(src, maskedSrc, prevChar)) {
```

Comment:
This would definitely be a breaking change since the tokenizers are part of the public API. Can we do this without combining them? Can we just switch the order of em and strong to get `<strong><em>a</em></strong>` to switch to `<em><strong>a</strong></em>`?

Reply:
Unfortunately, they kind of need to be tackled together to get the right sequence of `<em><strong>` to work, which isn't just a stylistic thing. Even though it renders the same as `<strong><em>`, the processing to get to that point also clears up several other bugs, especially regarding uneven `**text*****` delimiters on both sides.

Reply:
After researching quite a few dependants I think it should be fine to combine them since most dependants will change the renderer instead of the tokenizer. This will have to be a major bump to v2 though. I do want to get a few other breaking changes together before releasing v2 so it might be a while before I get to fully reviewing this PR.

Reply:
That sounds appropriate. I need to review the other PRs you have waiting as well that should go out before this anyway...

There are some other changes I've seen in the issues list that I'd like to lump into a v2 bump as well.